summary/percentiles.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Utility methods to calculate percentiles.
+
+package summary
+
+import (
+	"math"
+	"sort"
+
+	"github.com/golang/glog"
+	"github.com/google/cadvisor/info"
+)
+
+const milliSecondsToNanoSeconds = 1000000
+const secondsToMilliSeconds = 1000
```

Comment:
changed secondsToNanoseconds to be secondsToMilliseconds * milliSecondsToNanoseconds